lib/secure_headers/headers/content_security_policy.rb

```diff
@@ -13,6 +13,7 @@ class ContentSecurityPolicy
     FALLBACK_VERSION = ::UserAgent::Version.new("0")
 
     def initialize(config = nil, user_agent = OTHER)
+      user_agent ||= OTHER
```

Comment:
Since we are defaulting `user_agent` to `nil` in various places I thought it made sense to also override `user_agent` if `nil` ends up being passed.